deploy/tests/test_update_drones_etryvoga_v1.py

```
@@ -197,3 +199,39 @@ def mock_get_cache_data_side_effect(mc, key, default=None):
+    перевірка мапінгу
+    """
+
+    for _, region_data in regions.items():
+        mock_mc = AsyncMock(spec=Client)
+        mock_mc.set.return_value = True
+
+        def mock_get_cache_data_side_effect(mc, key, default=None):
+            mock_responses = {
+                b"drones_etryvoga": {
+                    "version": 1,
+                    "states": {str(region_data["id"]): {"lastUpdate": "2025-01-15T10:00:00Z"}},
```

Ruff (0.8.2)

Function definition does not bind loop variable `region_data`

(B023)